docs/source/etl/variant-normalization.rst

```diff
@@ -13,18 +13,24 @@ Variant Normalization
     df_original = spark.read.format('vcf').load(test_dir + 'test_left_align_hg38_altered.vcf')
     ref_genome_path = test_dir + 'Homo_sapiens_assembly38.20.21_altered.fasta'
 
-Different genomic analysis tools often represent the same genomic variant in different ways, making it non-trivial to integrate and compare variants across call sets. Therefore, **variant normalization** is an essential step on variants before further downstream analysis to make sure the same variant is represented identically in different call sets. Normalization is achieved by making sure the variant is parsimonious and left-aligned (see `Variant Normalization <https://genome.sph.umich.edu/wiki/Variant_Normalization>`_ for more details).
+Different genomic analysis tools often represent the same genomic variant in different ways, making it non-trivial to integrate and compare variants across call sets. Therefore, **variant normalization** is an essential step to be applied on variants before further downstream analysis to make sure the same variant is represented identically in different call sets. Normalization is achieved by making sure the variant is parsimonious and left-aligned (see `Variant Normalization <https://genome.sph.umich.edu/wiki/Variant_Normalization>`_ for more details).
 
-Glow provides the ``normalize_variants`` transformer to be applied on a variant DataFrame to normalize its variants, bringing unprecedented scalability to this operation. When applied on an input DataFrame of variants (e.g., generated by loading VCF or BGEN files), this transformer generates a DataFrame containing normalized variants.
+Glow provides variant normalization capabilities as a DataFrame transformer as well as a SQL expression function with Python and Scala APIs, bringing unprecedented scalability to this operation.
```

Comment:
Let's leave out the Scala part for now, as we don't have Scala API documentation yet. `Glow provides variant normalization capabilities as a DataFrame transformer as well as a SQL expression function with Python and Scala APIs, bringing unprecedented scalability to this operation.` -> `Glow provides variant normalization capabilities as a DataFrame transformer as well as a SQL expression function with a Python API, bringing unprecedented scalability to this operation.`

Reply:
Done